github/messages.go

```diff
@@ -56,6 +56,7 @@ var (
 		"page_build":                            "PageBuildEvent",
 		"public":                                "PublicEvent",
 		"pull_request_review_comment":           "PullRequestReviewCommentEvent",
+		"pull_request_review":                   "PullRequestReviewEvent",
```

Comment:
For consistency with elsewhere, you can move `pull_request_review` on top of `pull_request_review_comment`.